public/client.js

```diff
@@ -8,12 +8,26 @@ function addProduct() {
   userRequest.send(JSON.stringify({'product':document.getElementById("addProduct").value, 'quantity': document.getElementById("quantity").value}));
 }
 
-function addProductFromSearch() {
+function addProductFromSearch(e) {
   console.log("Adding a product...")
+  
+  var q = document.getElementById("quantity").value;
+  if (q==""){
+    alert ("Please specify a quantity");
+    return false;
```

Comment:
Does this also catch if you enter a non-number value?

Reply:
nice catch! thanks Nicole!
I'll make a note for next sprint as i'm having troubles with my local repository for this branch. I'll definitely add that test later.